### PR TITLE
propose text to address JS review

### PR DESCRIPTION
**Pull request by saradickinson**
Inclusion of the use of a 'timeout' in new text needs review....

### CODE REVIEW

File: draft-ietf-dprive-dnsoquic.md [hunk 1]
```diff
@@ -302,9 +302,15 @@ This means that the client's first query occurs on QUIC stream 0, the second on
...
+on the stream selected by the client.
+
+Servers and clients MAY monitor the number
+of "dangling" streams. These are open streams for which
+
+* the expected queries or responses have not been received or,
+* the expected queries or responses have been received but not the STREAM FIN
+
+after implementation defined timeouts. Implementations MAY impose a limit on the number of
 such dangling streams. If limits are encountered, implementations MAY close the connection.
```
Comment (huitema) [inline draft-ietf-dprive-dnsoquic.md:315]: I am not too keen on having the qualifier "after implementation defined timeouts" arriving at the beginning of the paragraph following the two bullets.

**Review by huitema — APPROVED** [review]
Looks good, but see whether there is a way to arrange the text and not have a dangling qualifier after the bullets defining dangling...

File: draft-ietf-dprive-dnsoquic.md (hunk 1 above)
Comment (saradickinson) [reply draft-ietf-dprive-dnsoquic.md:316]: Yeah - I couldn't get this quite right.... Any better now?

Comment (huitema) [reply draft-ietf-dprive-dnsoquic.md:316]: Yes, looks good to me.

**Review by huitema — APPROVED** [review]

**Pull request merged**